### ISSUE TITLE
Video is broken on older browsers

### ISSUE THREAD

**Issue by 12** [bbc/simorgh#2498]
**Describe the bug**
Video does not render on browsers that do not support arrow functions.

_But why are there arrow functions in the first place, do we not transpile?_
We do transpile, however these functions are as strings that are injected into the `head` via helmet:

https://github.com/bbc/simorgh/blob/0df6465e69241433dabac1a0be71a757380048e1/src/app/components/AudioVideoHead/index.jsx#L23-L47

Also note that we are calling `mediaPlayerSetup` with no parameters, despite the function definition containing a `container` parameter. This should also be fixed.

**To Reproduce**

1. Use an old browser that does not support arrow functions (this was spotted on Safari on an iPhone 5)
2. Go to https://www.test.bbc.co.uk/news/articles/c3wmq4d1y3wo
3. Video will not render
4. See error in console

**Expected behavior**
When using an older, but still supported browser, video should render with no errors in the console.

**Screenshots**
![image](https://user-images.githubusercontent.com/7690572/61134782-416b4e80-a4b8-11e9-8e53-3a0d15dd82cc.png)

**iPhone 5:**
- OS: iOS
- Browser: Safari
- Version: iOS 8.2

**Additional context**
The above snippet of code should be pure JS.

**Comment by twinlensreflex** [comment]
blocked on https://github.com/bbc/simorgh/issues/2545

**Comment by 12** [comment]
This is no longer relevant as our approach to video has changed, which mitigates this issue entirely.
